### ISSUE TITLE
Converting Prediction to String

### ISSUE THREAD

**Issue by Gisgar3** [tensorflow/tfjs#589]
To get help from the community, check out our [Google group](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs).

#### TensorFlow.js version
Latest
#### Browser version
*Not in browser; Node.JS*
#### Describe the problem or feature request
I am trying to convert my ``model.predict`` into a string, but am unsure how to do so. I know that ``.print()`` allows me to print my **tensor2d** into the console, but I am trying to convert the single value that the prediction outputs into a string variable. How would I go about this? Thanks!
#### Code to reproduce the bug / link to feature request
*N/A*

**Comment by bileschi** [comment]
Currently the standard way to do this is to find the arg-max of your logit layer, and then use the index into a map from integer class-indices to string class-labels.

You can see this pattern in the mobilenet example here:
https://github.com/tensorflow/tfjs-examples/blob/master/mobilenet/index.js#L95
